docs/examples/huggingface/README.md

```diff
@@ -199,6 +199,46 @@ inference_request = {
 requests.post("http://localhost:8080/v2/models/transformer/infer", json=inference_request).json()
 ```
 
+### Masked Language Modeling
```

Comment:
What's the connection between masked language models and Japanese support?

Reply:
There isn't any explicit connection. Apologies if I misinterpreted your earlier comment. I was showcasing this support by example like the other tasks in this section. If there's somewhere else we should explicitly document language support, I'd be happy to add something there.

Reply:
Also just for clarification, the language support doesn't require anything other than installing the optional dependencies. We're using a mask fill example to showcase this because it's a common way language models are pretrained. The point of the test is that some model (in the case a text fill model) can tokenize and make an inference on Japanese text.

Reply:
Maybe the heading should reflect that it is an example for using an optional library that is required for tokenizers?